src/features/amUI/common/PageContainer/PageContainer.tsx

```
@@ -2,9 +2,12 @@ import React from 'react';
 import { ArrowLeftIcon } from '@navikt/aksel-icons';
 import { useTranslation } from 'react-i18next';
 import { Link as DSLink } from '@digdir/designsystemet-react';
-import classes from './PageContainer.module.css';
 import { Link } from 'react-router';
-import { Button } from '@altinn/altinn-components';
+import { Button, DsLink } from '@altinn/altinn-components';
+
+import classes from './PageContainer.module.css';
+
+import classes from './PageContainer.module.css';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'classes'. Consider to delete it or rename it.

'classes' is defined here:


(lint/suspicious/noRedeclare)